No future requests

ruberoyce opened the pull request:
Fixes #7

ruberoyce committed:
Add a test script to package.json

ruberoyce committed:
Require that the request is for an execution block in the past

ruberoyce committed:
Add tests for only new requests with past execution block

0xbunnygirl commented on contracts/RequestForReorg.sol:
```diff
@@ -27,6 +27,7 @@ contract RequestForReorg {
     /// @param expiryBlock is the block at which the request expires
     function request(uint48 executeBlock, uint48 expiryBlock) external payable {
         require(requests[msg.sender].expiryBlock == 0, "Existing request");
+        require(executeBlock < block.number, "executeBlock must be in the past");
```
Think this needs to be `executeBlock <= block.number` because the miner would include the signed `request` tx before the `reorg` tx to replay the request

ruberoyce replied:
great point

ruberoyce committed:
Allow requests to be created with execution block right now

To support when the reorg happens, and miner includes the request tx into their reorg block
